src/screens/WhotoFollow/index.js

```diff
@@ -345,6 +347,20 @@ const styles = StyleSheet.create({
     marginBottom: dimen.normalizeDimen(24),
     paddingHorizontal: dimen.normalizeDimen(20)
   },
+  textShowMore: {
+    fontFamily: 'Inter-Medium',
+    fontWeight: '500',
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(20),
+    color: COLORS.signed_primary,
+    marginLeft: dimen.normalizeDimen(16),
+    marginBottom: dimen.normalizeDimen(16),
+    marginTop: dimen.normalizeDimen(8)
+  },
+  textShowMoreBold: {
+    fontFamily: 'Inter-Bold',
+    fontWeight: '700'
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of new styles `textShowMore` and `textShowMoreBold` indicates that the developer has added a "Show more" button to each section in the list. This is a good UX practice as it allows users to load more items in a specific section if they wish.
```diff
+  textShowMore: {
+    fontFamily: 'Inter-Medium',
+    fontWeight: '500',
+    fontSize: normalizeFontSize(14),
+    lineHeight: normalizeFontSize(20),
+    color: COLORS.signed_primary,
+    marginLeft: dimen.normalizeDimen(16),
+    marginBottom: dimen.normalizeDimen(16),
+    marginTop: dimen.normalizeDimen(8)
+  },
+  textShowMoreBold: {
+    fontFamily: 'Inter-Bold',
+    fontWeight: '700'
+  },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->